bin/mozilla/aa.pl

```diff
@@ -1896,6 +1896,7 @@ sub search {
 	      $vc
 	      $invnumber
           $currency
+          <tr>
```

Comment:
@ledger123 I'm not quite sure if it is the right place, or if it should be before $vc. But I guess it should be here, didn't know where it is displayed.

Reply:
This was added by you. Right. We need to remove this from here and then change <tr> to </tr> on line 1811 of develop as of now. I am doing both things.